### DISCUSSION TITLE
Extension no longer working at Facebook

### DISCUSSION THREAD

**Discussion by rcarter515** [extesy/hoverzoom#1146] [Q&A]
Very recently noticed that extension stopped working at Facebook.  I tried using Edge, Vivaldi (default), and Google Chrome.  HoverZoom works all other expected sites.  Version reported is Version 1.0.205 (last update: Tue, May 16, 2023).  

Checked settings and even uninstall/reinstall.

**Comment by extesy** [comment]
Already reported: https://github.com/extesy/hoverzoom/issues/1145. Facebook changed their design so the extension needs to be updated to support it.
